src/lsm/groove.zig

```diff
@@ -1434,47 +1434,3 @@ pub fn GrooveType(
         }
     };
 }
-
-test "Groove" {
-    const Transfer = @import("../tigerbeetle.zig").Transfer;
-    const IO = @import("../io.zig").IO;
-    const Storage = @import("../storage.zig").StorageType(IO);
-
-    const Groove = GrooveType(
-        Storage,
-        Transfer,
-        .{
-            .ids = .{
-                .timestamp = 1,
-                .id = 2,
-                .debit_account_id = 3,
-                .credit_account_id = 4,
-                .pending_id = 5,
-                .timeout = 6,
-                .ledger = 7,
-                .code = 8,
-                .amount = 9,
-            },
-            // Doesn't matter for this test.
-            .batch_value_count_max = .{
-                .timestamp = 1,
-                .id = 1,
-                .debit_account_id = 1,
-                .credit_account_id = 1,
-                .pending_id = 1,
-                .timeout = 1,
-                .ledger = 1,
-                .code = 1,
-                .amount = 1,
-            },
-            .ignored = [_][]const u8{ "user_data_128", "user_data_64", "user_data_32", "flags" },
-            .optional = &[_][]const u8{},
-            .derived = .{},
-            .orphaned_ids = true,
-        },
-    );
-
-    std.testing.refAllDecls(Groove);
-    std.testing.refAllDecls(Groove.PrefetchWorker);
-    std.testing.refAllDecls(Groove.PrefetchContext);
-}
```

Comment:
Figured that I'd rather delete these kinds of test, as I don't think they are bring value relative to zig build check.